airbyte-commons-worker/src/main/java/io/airbyte/workers/process/KubePodProcess.java

```diff
@@ -652,15 +637,6 @@ public int waitFor() throws InterruptedException {
     return exitValue();
   }
 
-  /**
-   * Waits for the Kube Pod backing this process and returns the exit value until a timeout. Closes
-   * resources if and only if the timeout is not reached.
-   */
-  @Override
-  public boolean waitFor(final long timeout, final TimeUnit unit) throws InterruptedException {
```

Comment:
what happened to this?

Reply:
@davinchia Our implementation just called the super class's implementation, which is in the `java.lang.Process` class.  Therefore, once the inheritance was removed, we no longer need this as we can just allow the implementation in `java.lang.Process` to stand in the new object we return from the `toProcess()` method.

Reply:
got it!